node/subsystem/src/messages.rs

```diff
@@ -389,6 +403,11 @@ pub enum RuntimeApiRequest {
 	/// Get all events concerning candidates (backing, inclusion, time-out) in the parent of
 	/// the block in whose state this request is executed.
 	CandidateEvents(RuntimeApiSender<Vec<CandidateEvent>>),
+	/// Get the `AuthorityDiscoveryId`s corresponding to the given `ValidatorId`s.
+	/// Currently this request is limited to validators in the current session. 
```

Comment:
I see, then let's ignore the session index elsewhere for now as is currently done.

Reply:
Yeah, what I'm worried about though is breaking changes. It's not clear how to make the API future-proof though.